common/app/model/content.scala

```diff
@@ -276,6 +276,18 @@ final case class Content(
       ("contributorBio", JsString(contributorBio.getOrElse(""))),
     )
 
+  def cricketTeam: Option[String] = {
+    if (tags.isCricketLiveBlog && conf.switches.Switches.CricketScoresSwitch.isSwitchedOn) {
+      CricketTeams.teamFor(this).map(_.wordsForUrl)
+    } else None
+  }
+
+  def cricketMatchDate: Option[String] = {
+    if (tags.isCricketLiveBlog && conf.switches.Switches.CricketScoresSwitch.isSwitchedOn) {
+      Some(trail.webPublicationDate.withZone(DateTimeZone.UTC).toString("yyyy-MM-dd"))
+    } else None
+  }
```

Comment:
Making the cricket data available on `ContentType` removes the complexity of going through the JsConfig to get the data!  Thanks @jamesgorrie